corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -0,0 +1,74 @@
+import urllib2, urllib
+import re
+from django.conf import settings
+from corehq.messaging.smsbackends.http.models import SQLSMSBackend
+from corehq.messaging.smsbackends.vertex.forms import VertexBackendForm
+
+VERTEX_URL = "http://www.smsjust.com/sms/user/urlsms.php"
```

Comment:
their certificate is not great. It still needs a manual confirmation when accessed from browser and fails with SSLError from code.
```
SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",)
```